.gitignore

```diff
@@ -124,4 +125,4 @@ ios/vendor
 i18n-test/
 
 # i18n cache
-src/i18n-cache/
\ No newline at end of file
+src/i18n-cache/
```

Comment:
Nit and not a blocker for LGTM'ing the PR: what do you think about undoing the diff of the file ending, or separating it into a commit of its own to help with code archeology? :)